persistence-modules/spring-boot-persistence-h2/src/test/java/com/baeldung/ItemIntegrationTest.java

```diff
@@ -0,0 +1,31 @@
+package com.baeldung;
+
+import com.baeldung.h2db.springboot.SpringBootH2Application;
+import com.baeldung.h2db.springboot.daos.ItemRepository;
+import com.baeldung.h2db.springboot.models.Item;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = SpringBootH2Application.class)
+public class ItemIntegrationTest {
+
+    @Autowired
+    private ItemRepository itemRepository;
+
+    @Test
+    public void shouldNotAllowToPersistNullItemsPrice() {
+        Item saved = null;
+        try {
+            saved = itemRepository.save(new Item());
+        } catch (Exception e) {
```

Comment:
This test will go green if no exception is thrown, check this method:
https://joel-costigliola.github.io/assertj/core-8/api/org/assertj/core/api/Assertions.html#assertThatThrownBy-org.assertj.core.api.ThrowableAssert.ThrowingCallable-